test/Number.js

```diff
@@ -0,0 +1,23 @@
+// Prepare chai
+const chai = require('chai');
+const expect = chai.expect;
+
+describe('Number', function() {
+  describe('static isNumber', function() {
```

Comment:
If you are describing static functions then please use `#`. E.g. `#isNumber`. When you are describing instance method then use dot - `.something`

Reply:
Thanks, I'll fix it.